lib/trie_ascoltatore.js

```diff
@@ -50,11 +51,14 @@ TrieAscoltatore.prototype.publish = function (topic, message, options, done) {
 
   var cbs = this._matcher.match(topic);
 
-  for (var i = 0; i < cbs.length; i++) {
-    cbs[i](topic, message, options);
+  function clientCallback(cb, next){
+    setImmediate(function(){
+      cb(topic, message, options);
+    });
+    next();
```

Comment:
Considering a huge (multi hundred thousands) list of forwarder callbacks, Does't `setImmediate` improve the iteration?Or steed does the same job?